EIPS/eip-FOCIL.md

```diff
@@ -40,12 +40,55 @@ TBD
 A set of validators is selected from the beacon committee to become IL committee for `slot N`.
 
 - **`Slot N`, `t=0 to 8s`**: After processing the block for `slot N` and confirming it as the head, each IL committee member of `slot N` constructs a local inclusion list based on the head and their view of the public mempool, then broadcasts it over the P2P network.
```

Comment:
What happens between 8s and 9s? Can a IL committee member broadcast a new and first local IL?

Reply:
No an IL committee member should stop broadcasting new local ILs at  8s